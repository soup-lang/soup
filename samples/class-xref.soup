class Fun
    func get()
        console.log('Fun::get')
        return Gun()

class Gun
    func get()
        console.log('Gun::get')
        return Fun()

f: Fun()
g: f.get()
h: g.get()
h.get()
