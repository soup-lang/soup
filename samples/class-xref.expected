Fun::get
Gun::get
Fun::get
